vendors/cypress/boards/CYW943907AEVAL1F/aws_tests/config_files/aws_test_runner_config.h

```diff
@@ -50,6 +50,8 @@
 #define testrunnerFULL_GGD_HELPER_ENABLED             0
 #define testrunnerFULL_MQTT_AGENT_ENABLED             0
 #define testrunnerFULL_MQTT_ALPN_ENABLED              0
+#define testrunnerFULL_CORE_MQTT_ENABLED              0
+#define testrunnerFULL_CORE_MQTT_ENABLED
```

Comment:
What is the purpose of defining this twice?

Reply:
That was accidental. Fixed